### ISSUE TITLE
NTR: Eyes: multiple beaded papules along the eyelid margins and inner canthus (Eye structure & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7991]
New term request Eyes: multiple beaded papules along the eyelid margins and inner canthus (elthat_160125160046): Blepharosis  UMLS: 3550510

**Comment by pnrobinson** [comment]
@MickeySegal
is this distinct from
[Multiple eyelid margin cysts](https://hpo.jax.org/app/browse/term/HP:0034946)?

**Comment by MickeySegal** [comment]
These are papules:
https://www.ncbi.nlm.nih.gov/books/NBK338540/
Moniliform blepharosis (the presence of multiple beaded papules along the eyelid margins and inner canthus; see [Figure 1d](https://www.ncbi.nlm.nih.gov/books/NBK338540/figure/lipoid-p.F1/?report=objectonly), [1e](https://www.ncbi.nlm.nih.gov/books/NBK338540/figure/lipoid-p.F1/?report=objectonly)), is a pathognomic sign

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000842
- label: Multiple eyelid beaded papules
- parents: [Abnormal eyelid morphology (HP:0000492)](https://hpo.jax.org/app/browse/term/HP:0000492)
- pmid: PMID:26655007
- def: Multiple beaded papules along the eyelid margins and inner canthus.
- comment: See Figure 2 of PMID:26655007.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7991
- synonyms: Moniliform blepharosis(NONE)
